po/zh_CN.po

```diff
@@ -2,81 +2,80 @@
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
 # 小高 <2926295173@qq.com>, 2021.
+# TianXing Yi <duguqiubailee@gmail.com>, 2021.
 msgid ""
 msgstr ""
 "Project-Id-Version: gnucash-htdocs r18948\n"
-"Report-Msgid-Bugs-To: https://bugs.gnucash.org/buglist.cgi?"
-"component=Translations&product=Website&resolution=---\n"
+"Report-Msgid-Bugs-To: https://bugs.gnucash.org/buglist."
+"cgi?component=Translations&product=Website&resolution=---\n"
 "POT-Creation-Date: 2021-03-20 07:48+0100\n"
-"PO-Revision-Date: 2021-03-03 08:50+0000\n"
-"Last-Translator: 小高 <2926295173@qq.com>\n"
+"PO-Revision-Date: 2021-03-31 03:26+0000\n"
+"Last-Translator: TianXing Yi <duguqiubailee@gmail.com>\n"
 "Language-Team: Chinese (Simplified) <https://hosted.weblate.org/projects/"
 "gnucash/website/zh_Hans/>\n"
 "Language: zh_CN\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Weblate 4.5.1-dev\n"
+"X-Generator: Weblate 4.6-dev\n"
 
 #: 2.6-release-tour.phtml:3
-#, fuzzy
 msgid "GnuCash 2.6 release tour"
-msgstr "GnuCash %s 运行于"
+msgstr "GnuCash 2.6发布之旅"
 
 #: 2.6-release-tour.phtml:4
 msgid ""
 "This tour will introduce you to new major features in <span class=\"gnucash"
 "\">GnuCash</span> 2.6."
-msgstr ""
+msgstr "此导览将向您介绍 GnuCash 2.6 新增的主要功能。"
```

Comment:
@duguqiubailee I am missing the html markup `<span class=\"gnucash""\">…</span>`.

Reply:
Hi, I have realized it and have corrected.

Sorry for the first time about translation, I will do it more carefully later.

And will pay more attention to github’s notifications.

Thanks for your patience and help.